### ISSUE TITLE
Single short lines in the middle of a file are not detected

### ISSUE THREAD

**Issue by mdshw5** [mdshw5/pyfaidx#44]
```
>AB821309.1
ATGGTCAGCTGGGGTCGTTTCATCTGCCTGGTCGTGGTCACCATGGCAACCTTGTCCCTGGCCCGGCCCT
CCTTCAGTTTAGTTGAGGATACCACATTAGAGCCAGAAGATGCCATCTCATCCGGAGATGATGAGA
CACCGATGGTGCGGAAGATTTTGTCAGTGAGAACAGTAACAACAAGAGAGCACCATACTGGACCAACACA
```

does not raise an error

**Comment by mdshw5** [comment]
Currently produces"
```
(py34)LT-SHIRLEYM:data matt$ faidx short_line.fasta
>AB821309.1
ATGGTCAGCTGGGGTCGTTTCATCTGCCTGGTCGTGGTCACCATGGCAACCTTGTCCCTGGCCCGGCCCT
CCTTCAGTTTAGTTGAGGATACCACATTAGAGCCAGAAGATGCCATCTCATCCGGAGATGATGAGACACC
GATGGTGCGGAAGATTTTGTCAGTGAGAACAGTAACAACAAGAGAGCACCATACTGGACCAACACA
```

**Comment by mdshw5** [comment]
Fixed in 48dd0701c457c808a251ccbaf4cd192f1382b99b.
